packages/client/src/checkout/types/getCheckoutSession.types.ts

```diff
@@ -0,0 +1,7 @@
+import type { CheckoutSession } from './checkoutSession.types.js';
+import type { Config } from '../../types/index.js';
+
+export type GetCheckoutSession = (
+  checkoutOrderId: CheckoutSession['id'],
```

Comment:
```suggestion
  checkoutSessionId: CheckoutSession['id'],
```

I believe you meant `checkoutSessionId` - is the name of the argument and the same as in `GetCheckoutSessionCharge`

Reply:
I meant indeed, thanks for catching that!